cdm/src/main/java/ucar/nc2/ft2/coverage/simpgeometry/SimpleGeometryReader.java

```diff
@@ -61,30 +73,24 @@ public Point readPoint(String name, int index) {
 
 		Variable pointvar = ds.findVariable(name);
 		if(pointvar == null) return null;
+		Point pt = null;
 		
-		Point pt = new CFPoint(-1, -1, null, null, null);
-		return pt.setupPoint(ds, pointvar, index);
+		// CFConvention
+		if(ds.findGlobalAttribute(CF.CONVENTIONS) != null)
+			if(ucar.nc2.dataset.conv.CF1Convention.getVersion(ds.findGlobalAttribute(CF.CONVENTIONS).getStringValue()) >= 0)
+				pt = new CFPoint();
+		
+		if(pt == null) return pt;
+		else return pt.setupPoint(ds, pointvar, index);
 	}
 	
 	/**
 	 * Constructs a new Simple Geometry Reader over the specified dataset.
-	 * Assumes the CF Convention.
 	 * 
 	 * @param ds - the specified dataset
 	 */
 	public SimpleGeometryReader(NetcdfDataset ds) {
 		this.ds = ds;
 	}
 	

```

Comment:
Good call on eliminating this constructor. I think it makes sense to assume CF.

Reply:
Thanks, it also allows for being extended to other standards as well!